core-web/libs/ui/src/lib/components/dot-workflow-actions/dot-workflow-actions.component.html

```diff
@@ -4,7 +4,7 @@
     @if (subActions.length) {
         <p-splitButton
             (onClick)="mainAction?.command({ originalEvent: $event })"
-            [disabled]="loading()"
+            [disabled]="loading() || disabled()"
```

Comment:
two "states" for the same action?

Reply:
They are the same action because the `splitButton` does not have the `loading` state - the normal `pButton` has both -.

If the `WF` actions are loading, we should disable the `splitButton` to prevent user clicks.

More information: https://primeng.org/splitbutton